playbooks/cluster/openshift/vars/gluster.yml

```diff
@@ -0,0 +1,8 @@
+# Namespace for CNS pods (will be created)
+openshift_storage_glusterfs_namespace: app-storage
+# Automatically create a StorageClass referencing this CNS cluster
```

Comment:
this? which one? 
CNS cluster? clusterS or podS as from the line earlier?

'Automatically' is redundant, because if it's an option true/false, that causes the storage class to be created automatically.